src/components/UserList/UserList.js

```diff
@@ -44,6 +44,7 @@ class UserList extends Component {
         }),
       });
 
+      result[result.length - 1].last = true;
```

Comment:
To give margin right to last card, I wrote this code.
You can find a code giving margin in Row.js